openmc/universe.py

```diff
@@ -611,6 +616,14 @@ def _determine_paths(self, path='', instances_only=False):
             if not instances_only:
                 cell._paths.append(cell_path)
 
+    def _deepcopy_universe_for_clone(self):
+        """Clone all of the openmc.Universe object's attributes except for its cells, as they will be handled within the clone function.
+        This should only to be used within the openmc.UniverseBase.clone() context and is more performant than a deepcopy.
+        """
```

Comment:
I would perhaps rename to `_partial_deepcopy`. Also, please reformat the docstring here (and below) to fit within 80 columns.